doc/concepts/data-concepts/provenance.md

```diff
@@ -0,0 +1,67 @@
+# Provenance
+
+Provenance enables Pachyderm users to go back in time and see the state of
+data at a particular moment in the past. Data scientists use provenance in
+root cause analysis, as well as to improve their software code.
+Provenance tracks the
+dependency between datasets and determines their origins. Therefore,
+provenance answers not only the question of where the data comes from,
+but also how the data was transformed along the way. Data scientists need
+to have confidence in the information with which they operate. They need
+to be able to reproduce the results and sometimes go through the whole
+data transformation process from scratch multiple times, which makes data
+provenance one of the most critical aspects of data analysis. If your
+computations result in unexpected numbers, the first place to look
+is the historical data that gives insights into possible flaws in the
+transformation chain or the data itself.
+
+When a bank makes a decision about a mortgage
+application, many factors are taken into consideration, including the credit
+credit history, annual income, and loan size. This data goes through multiple
+automated steps of analysis with numerous dependencies and decisions made
+along the way. If the final decision does not satisfy the applicant,
+the historical data is the first place to look for proof of authenticity,
+as well as for possible prejudice against the applicant. Data provenance
+enables data scientists to track the
+progress from its origin to the final decision and make appropriate
+changes that address issues. With the adoption of General Data
+Protection Regulation (GDPR) compliance requirements, monitoring data lineage
+is becoming a necessity for many organizations that work with sensitive data.
+
+Pachyderm implements provenance for both commits and repositories.
+Therefore, you can track revisions within one branch and
+understand the connection between the data stored in one repository
+and the data that was used to calculate the result in the other
+repository. Therefore, you can track the data transformation process across
+multiple datasets.
+
+Collaboration takes data provenance even further. You can make any dataset
```

Comment:
> You can make any dataset
available to other members of your team. When many data scientists have
access to the same dataset, they can conduct their own experiments with
the data and identify better data analysis processes by using separate branches

This sentence is true, but doesn't really fit the idea of provenance. This is about branches. I think like below makes more sense:

"Data provenance takes collaboration even further. Provenance allows teams across the world to share, transform, and update data sets while automatically maintaining a complete audit trail so everything is 100% reproducible."